src/schemas/v2/pdtf-transaction.json

```diff
@@ -33248,6 +33248,39 @@
               }
             ]
           }
+        },
+        "documents": {
+          "title": "Documents",
+          "description": "Documents associated with the transaction",
+          "type": "array",
+          "items": {
+            "type": "object",
+            "properties": {
+              "displayName": {
+                "title": "Human-readable name of document",
+                "type": "string",
+                "minLength": 1
+              },
+              "fileName": {
+                "title": "File name of the document",
+                "type": "string",
+                "minLength": 1
+              },
+              "mimeType": {
+                "title": "Document MIME type",
+                "type": "string",
+                "minLength": 1
+              },
+              "documentType": {
+                "title": "Human-readable document category",
+                "type": "string",
+                "minLength": 1
+              },
+              "externalIds": {
```

Comment:
Why external ids are left as a object  this makes the schema values non deterministic, can we make external ids as list of strings. In place of just objects  .